src/lib/components/icons/checkmark.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+	import type { IconProps } from '$lib/types'
+
+	type $$Props = IconProps
+
+	let cls: string | undefined = undefined
+	export { cls as class }
```

Comment:
Just so you know, we should remove these. Adding class to a component is now done differently (and we don't need it)

Reply:
Oh cool, good to know! I'll remove it.